Example/Tests/Tests.m

```diff
@@ -359,6 +397,33 @@
             }];
         });
     });
+    
+    describe(@"video tracking", ^{
+        describe(@"initialization", ^{
+            SEGMockADBMediaHeartbeatFactory *mockADBMediaHeartbeatFactory = [[SEGMockADBMediaHeartbeatFactory alloc] init];
+            mockADBMediaHeartbeatFactory.ADBMediaHeartbeat = mockADBMediaHeartbeat;
+            integration = [[SEGAdobeIntegration alloc] initWithSettings:@{ @"heartbeatTrackingServer" : @"example" }
+                                                           andADBMobile:mockADBMobile andADBMediaHeartbeatFactory:mockADBMediaHeartbeatFactory andADBMediaHeartbeatConfig:mockConfig];
+            it(@"Video Playback Started initializes ADBMediaHeartbeat object", ^{
+                SEGTrackPayload *payload = [[SEGTrackPayload alloc] initWithEvent:@"Video Playback Started" properties:@{
+                                                                                                                         @"content_asset_id" : @"1234",
+                                                                                                                         @"ad_type" : @"pre-roll",
+                                                                                                                         @"video_player" : @"Netflix",
+                                                                                                                         @"channel" : @"Cartoon Network",
+                                                                                                                         @"": @""
+                                                                                                                         } context:@{}
+                                                                     integrations:@{@"Adobe Analytics": @{@"ovpName": @"Netflix", @"debug": @YES}}];
+                
+                [integration track:payload];
+                [verify(mockADBMediaHeartbeat) trackSessionStart:nil data:@{}];
+//                [verify(mockConfig.trackingServer) isEqualToString:@"example"];
```

Comment:
@f2prateek was hoping for some guidance here. Is there a simple way to setup the config to mock? Or do I have to set it up like the ADBMediaHeartbeat factory?

Reply:
Do we need a mockConfig? seems like it's just a dumb object - we should be able to use a real config object.

Reply:
Hmm how can we assert against those values? I thought we would need to pass in the same config object that is running through my logic.